src/cljs/main/broadfcui/common/entity_table.cljs

```diff
@@ -127,7 +129,12 @@
                                                   (str ": " (string/join ", " items)))))
                                          :else ((:attribute-renderer props) attr-value)))})
                           attributes))
-               :column-defaults (get (:column-defaults props) selected-entity-type)
+               :column-defaults (if (some? process-local-state)
+                                  (let [local-state (persistence/check-saved-state {:key (str (common/workspace-id->string (:workspace-id props)) ":data:" (:selected-entity-type @state))})]
+                                    (let [shown (mapv #(get % :id) (get (group-by :visible? (:column-display local-state)) true))
+                                          hidden (mapv #(get % :id) (get (group-by :visible? (:column-display local-state)) false))]
```

Comment:
Seems like you could let out `(get (group-by :visible? (:column-display local-state))`